task/scorecard/0.1/scorecard.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: scorecard
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: scorecard
+    tekton.dev/displayName: OSSF scorecard
+
+spec:
+  description: >-
+    This task can be used to calculate the scorecard results for a repository
+
+  params:
+    - name: REPO
+      description: The repository URL to inspect
+      type: string
+    - name: GITHUB-OAUTH_SECRET
+      description: The GitHub secret with credentials
+      type: string
+      default: "github"
+  steps:
+    - name: scorecard
+      image: gcr.io/openssf/scorecard:latest
```

Comment:
can we change the `latest` tag to `tag+digest` format

Reply:
Done!